fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneDocumentFromRecord.java

```diff
@@ -193,12 +193,14 @@ public void addField(@Nonnull T source, @Nonnull final String fieldName, @Nullab
         private final Object value;
         private final LuceneIndexExpressions.DocumentFieldType type;
         private final boolean stored;
+        private final Map<String, Object> fieldConfigs;
```

Comment:
Probably should mark this as `@Nullable` (unless it's made `@Nonnull` via setting it to the empty map if null)

Reply:
Use Collections.emptyMap() now